.github/workflows/check-pr-title.yaml

```diff
@@ -0,0 +1,12 @@
+name: 'Check PR title for good'
+
+on:
+  pull_request: {}
+
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check if the PR title matches the good PR regex
```

Comment:
I think we should also check the length of message. Probably 80 chars will be enough
I've create a histogram from commit message lengths (including PR number suffix)
![image](https://github.com/stackrox/stackrox/assets/1616386/88a969cd-14fc-4283-b424-9fb84d12cd89)

Reply:
I find 80 too limiting. If set a limit, I'd think of 100 or 120.

Consider this (91 chars):
> ROX-7197: explicitly return nil and log when service account is missing for the deployment